content/guides/virtualizing/kvm.md

```diff
@@ -37,94 +36,80 @@ As of Ubuntu 18.10, the `libvirt-bin` package has been replaced, so the necessar
-
-![](/files/guides/virtualizing/kvm/virtual_network_1.png)
-
-Name the virtual network that will be created.
-
-![](/files/guides/virtualizing/kvm/net_name.png)
-
-Here, you can customize the network address range. If you are fine with the default settings, simply click **`Forward`**
-
-![](/files/guides/virtualizing/kvm/ip_range.png)
-
-If you want to enable the IPv6 address range, you can customize it here. Otherwise, just click **`Forward`**
-
-![](/files/guides/virtualizing/kvm/ipv6.png)
-
-Here, you can forward the network packets of the VM to a physical network, or keep the virtual network isolated. If you need Internet connection on the VM, you should forward to a physical network connected to the Internet. When you are done, click **`Finish`**
-
-![](/files/guides/virtualizing/kvm/connect_physical.png)
-
-The virtual network will then be created. Now, close the Connection Details window.
-
-On the virt-manager window, right-click on the Haiku VM, and choose **`Open`**. Then, click on the button with the light-bulb icon to switch to the VM configuration tab.
+![step_five](/assets/step_five_ky53f4k8m.png)
 
-![](/files/guides/virtualizing/kvm/vm_settings.png)
-
-If there's no NIC hardware in the VM, click the **`Add Hardware`**, then choose **`Network`**.
-
-After that, go to the **`NIC`** hardware on the VM, then choose the network source to be the virtual network we have created.
-
-![](/files/guides/virtualizing/kvm/vm_settings_net.png)
-
-Now the VM should have a network connection.
+After clicking **`Finish`**, a popup saying `Virtual Network is not active` may appear, don't worry, it's just asking you to enable the ``'default'`` network which is set by default, just click `yes` and everything should be fine, if not, see the [Troubleshooting section](#part_trouble). the VM will start and boot to the Haiku image. You can choose to install Haiku or boot to the desktop. Installation is simple and does not differ really from a physical one (follow the guides [on this page](/get-haiku/installation-guide) if you are not familiar with installing Haiku).
 
 ### Troubleshooting <a name="part_trouble"></a>
 
-##### Unable to connect to Libvirt
-
-If, after running Virt-manager, you run into a pop-up saying that it is unable to connect to libvirt, then:
+##### Unable to complete install
 
-1. Make sure the libvirtd daemon is running by issuing ```sudo systemctl start libvirtd```
+After finishing the installation of your VM If you run into an error that says **``unable to complete install: network default is not active``** or something along those lines, make sure that `libvirtd` is running by entering ``sudo systemctl start libvirtd`` in terminal, if the error still occurs do the following steps:
 
-2. Make sure you are a member of the 'libvirt' group by running: 
+>.1 Check if you have the network defined by running:
 ```sh
```

Comment:
We sometimes have markdown formatting trouble on our website. Make sure here and everywhere following, you have an empty line before and after the "sh" code block.